### DISCUSSION TITLE
Help with installation nvidia driver. Quadro k3000m old laptop.

### DISCUSSION THREAD

**Discussion by LNNRT-STN** [lllyasviel/Fooocus#1612] [Q&A] [answered]
I am encoutering this in my console when generating an image.

> RuntimeError: The NVIDIA driver on your system is too old (found version 8000). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver.    

I have a Nvidia Quadro K3000M on a dell precision m6700 (it is an old laptop of 2012)

> Intel(R) Core(TM) i7-3940XM CPU @ 3.00GHz   3.20 GHz
> 32,0 GB

I have the "latest" nvidia driver 22.21.13.8216, but it's from 2017 is there anything I can do?

I have first tried it on linux ubuntu and trying to get the cuda drivers working but couldn't manage to do so. I thaught on windows 10 it would work direcly :)

Thank you in advance

**Comment by Hypersonic-Popcorn** [comment, marked as answer]
I know this is a couple days old but in case you're still trying, [This driver](https://www.nvidia.com/download/driverResults.aspx/157254/en-us/) lists your GPU as supported and is considerably newer. You may need to find out what CUDA version it has and download a version of pytorch for it.

**Reply by LNNRT-STN** [reply]
Thank you for your answer, i am ecountering this error. I will try it in the future with a better gpu.

Have a nice day.

![image](https://github.com/lllyasviel/Fooocus/assets/117391122/ce46f365-6ed8-4485-9504-fbdbfcde9c8e)
